.github/workflows/issue-labeled.yml

```diff
@@ -0,0 +1,26 @@
+name: Issue Labeled
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  issue-labeled:
+    runs-on: ubuntu-latest
+    steps:
+      - name: blocked/need-repro
+        if: github.event.label.name == 'blocked/need-repro'
+        uses: actions-cool/issues-helper@v3
+        with:
+          actions: 'create-comment'
+          body: |
+            Hello @${{ github.event.issue.user.login }}. Thanks for reporting this and helping to make Electron better!
+
+            Because of time constraints, triaging code with third-party dependencies is usually not feasible for a small team like Electron's.
```

Comment:
Thanks so much for this PR! Could I ask you to remove this line from the comment? Not all of the blocked issues have a third-party dependency, so removing this will allow it to apply to a broader group of issues 🙇‍♀️

Reply:
👌🏻